src/ResourceManager/RecoveryServices.Backup/Commands.RecoveryServices.Backup.Helpers/Conversions/ConversionHelpers.cs

```diff
@@ -195,25 +195,33 @@ public static AzureRmRecoveryServicesBackupItemBase GetItemModel(ProtectedItemRe
                         HelperUtils.ParseUri(policyId);
                         policyName = HelperUtils.GetPolicyNameFromPolicyId(keyValueDict, policyId);
                     }
-                    itemModel = new AzureRmRecoveryServicesBackupIaasVmItem(protectedItem, container, policyName);
+
+                    string containerUri = HelperUtils.GetContainerUri(
+                        HelperUtils.ParseUri(protectedItem.Id),
+                        protectedItem.Id);
+
+                    itemModel = new AzureRmRecoveryServicesBackupIaasVmItem(
+                        protectedItem,
+                        IdUtils.GetNameFromUri(containerUri),
+                        Microsoft.Azure.Commands.RecoveryServices.Backup.Cmdlets.Models.ContainerType.AzureVM, 
```

Comment:
use an alias instead of  Microsoft.Azure.Commands.RecoveryServices.Backup.Cmdlets.Models.ContainerType.AzureVM